pkg/controller/controller.go

```diff
@@ -111,12 +146,21 @@ func StartLocalController(client *kubernetes.Clientset, ptable deleter.ProcTable
 	}
 	// Run controller logic.
 	if jobController != nil {
-		go jobController.Run(wait.NeverStop)
+		go jobController.Run(jobControllerStopChan)
```

Comment:
what would happen if the config is reloaded while a cleanup job is running?

Reply:
Summarizing in a table

| Old Config | New Config | Behavior
| ----------  | -----------  | ---------
| Job             | Job              | clean up done once using `Job`, job controller will just restart
| Job             | ProcTable   | clean up done twice on the volume
| ProcTable  | ProcTable    | reconstruct the clean up process to clean the volume again on restart
| ProcTable   | Job             | construct a `Job` resource to finish up the clean up of the volume

Reply:
Cool, thanks for the detailed explanation